taxonomy-editor-frontend/src/pages/editentry/LanguageSelectionDialog.tsx

```diff
@@ -52,15 +54,25 @@ const LanguageSelectionDialog = ({
               selected.map((langCode) => ISO6391.getName(langCode)).join(", ")
```

Comment:
This has to be modified as well, because currently when "All languages" is selected, it is rendered as an empty string:
![image](https://github.com/openfoodfacts/taxonomy-editor/assets/82757576/8c76acdf-755a-42d8-9eb8-029590cd9bec)